nemo/collections/nlp/modules/common/megatron/utils.py

```diff
@@ -375,10 +375,11 @@ def get_all_params_for_weight_decay_optimization(
     return ({'params': weight_decay_params},)
 
 
-def split_list(inputs, chunk_size):
+def split_list(inputs, num_chunks):
```

Comment:
will this break places other than our code?

Reply:
nope